systemd/status-mail-generator.sh.in

```diff
@@ -17,5 +17,5 @@ fi
 for service in ${SEND_ON_FAILURE}; do
     dropindir="${unitdir}/${service}.d/"
     mkdir -p "${dropindir}" || exit 1
-    ln -s @maildatadir@/status-mail.conf "${dropindir}"
+    ln -s /usr/share/systemd-status-mail/status-mail.conf "${dropindir}"
```

Comment:
This is wrong, path should not be fixed if they are configurable.
Why this change?